helm/cluster-api-bootstrap-provider-kubeadm/templates/service.yaml

```diff
@@ -11,10 +11,12 @@ metadata:
   namespace: {{ include "resource.default.namespace" . }}
```

Comment:
I see these annotations in other `Services`, I'd add them here for alignment sake
```
    prometheus.io/scrape: "true"
    giantswarm.io/monitoring: "true"
    giantswarm.io/monitoring-port: "{{ .Values.ports.metrics }}"
```

Reply:
Added